app/components/form/title_component.html.erb

```diff
@@ -1,14 +1,16 @@
 <div class="title-wrapper pt-30">
   <div class="title mb-30">
     <div class="align-items-baseline d-md-flex gap-3">
-      <h1><%= @title %></h1>
+      <h1 class="case-contacts-form desktop-title"><%= @title %></h1>
+      <h2 class="case-contacts-form mobile-title"><%= @title %></h2>
```

Comment:
I know the issue says to change "H1 to size H2", but we shouldn't change the semantic HTML to achieve that. Keep it is an H1 and just change the size of the text in the @media rule.